pkgs/applications/misc/go-jira/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "go-jira-0.1.14";
+
+  meta = {
+    description = "simple command line client for Atlassian's Jira service written in Go";
+    homepage = "https://github.com/Netflix-Skunkworks/go-jira";
+    licence = stdenv.lib.licenses.asl20;
+    platforms = stdenv.lib.platforms.linux;
```

Comment:
Good tip. I reckon with `go2nix` the platforms will change. I'll try to pull request with those changes later today. Thanks!